Sources/Datadog/FeaturesIntegration/CrashReporting/CrashReportingWithRUMIntegration.swift

```diff
@@ -78,6 +98,21 @@ internal struct CrashReportingWithRUMIntegration: CrashReportingIntegration {
         }
     }
 
+    /// If the crash occurred before starting RUM session (after initializing SDK, but before starting the first view) we don't have any session UUID to associate the error with.
+    /// In that situation, we create a dummy view to associate the error with. In result, a dummy RUM session will be started and it will contain one view and one error (crash).
+    private func sendCrashReportToNewDummySession(
```

Comment:
Would it be possible that this method and `sendCrashReportToPreviousSession` calls the same implementation so we don't duplicate the format? It could involve renaming the methods. WDYT?